speech/grpc/transcribe_streaming.py

```diff
@@ -64,7 +64,7 @@ def make_channel(host, port):
     return implementations.secure_channel(host, port, composite_channel)
 
 
-def _audio_data_generator(buff):
+def _audio_data_generator(buff, stoprequest):
```

Comment:
`_audio_data_generator` shouldn't need this. Once `_fill_buffer` gets the signal, it'll add `None` to the buffer, which will cause this generator to exit.